python.d/example.chart.py

```diff
@@ -0,0 +1,18 @@
+import random
+
+update_every = 5
+priority = 30000
```

Comment:
I would like this to be part of the existing `example` section.

Set this to `150000` (this controls the order of the charts on the dashboard).